server/index.js

```diff
@@ -213,6 +213,7 @@ if (helpers.isTestEnvironment()) {
   }
   server = https.createServer(httpsOptions, app).listen(8080)
   setInterval(vitals.checkHeartbeat, 60000)
+  setInterval(vitals.checkForInternalProblems, 24 * 60 * 60 * 1000) // check every 24 hours to see if internal alerts need to be sent 24 * 60 * 60 * 1000, change to a shorter value for texting purposes
```

Comment:
I wonder whether you want to make the 24 hours into an environment variable. It might make it easier for you to test because you'd be able to make it a smaller number in the dev environment. What do you think? If you do, then your DB function's name will need to change too so as not to reference 24 hours. Up to you!

Reply:
Oooh yes I considered doing that, I probably will do that!